umbral/pre.py

```diff
@@ -507,10 +470,19 @@ def _open_capsule(capsule: Capsule, bob_private_key: UmbralPrivateKey,
     """
     params = params if params is not None else default_params()
 
-    priv_b = bob_private_key.bn_key
-    pub_b = priv_b * params.g
+    priv_b = bob_privkey.bn_key
+    pub_b = bob_privkey.get_pubkey().point_key
 
-    pub_a = alice_pub_key.point_key
+    pub_a = alice_pubkey.point_key
+
+    # TODO: Change dict for a list if issue #116 goes through
```

Comment:
Is this an old TODO?

Reply:
No, I added it here in light of issue #116